ionicons/IonIcons.m

```diff
@@ -19,10 +19,33 @@ @implementation IonIcons
 + (UIFont*)fontWithSize:(CGFloat)size;
 {
     UIFont* font = [UIFont fontWithName:@"ionicons" size:size];
+    if (!font) {
+        [[self class] dynamicallyLoadFontNamed:@"ionicons"];
```

Comment:
ah nvm I see it doesn't really need dispatch_once because the next time around `font` will be non-nil the first time its creation is attempted.